### DISCUSSION TITLE
3D CPU Reconstruction

### DISCUSSION THREAD

**Discussion by geke03** [astra-toolbox/astra-toolbox#326] [Q&A]
Hello,

I use the Astra Toolbox on an AMD CPU notebook without a dedicated GPU. This means I can not use Cuda.

On the algorithm overview page:
https://www.astra-toolbox.com/docs/algs/index.html#id1
there is no 3D CPU algorithm listed.

Is it possible to do a 3D CPU Reconstruction with Astra Toolbox?

**Comment by scf819** [comment]
It seems that only 2d reconstruction of the cpu is supported？

**Comment by askorikov** [comment]
Hi! Yes, only 2D algorithms are implemented on CPU (as 3D versions would be prohibitively expensive for typical use cases).
